lib/actions/ssh.ts

```diff
@@ -145,15 +143,17 @@ function generateVpnSshCommand(opts: {
 	proxyUrl: string;
 	proxyCommand?: string;
 }) {
-	return (
-		`ssh ${
-			opts.verbose ? '-vvv' : ''
-		} -t -o LogLevel=ERROR -o StrictHostKeyChecking=no ` +
-		`-o UserKnownHostsFile=/dev/null ` +
-		`${opts.proxyCommand != null ? opts.proxyCommand : ''} ` +
-		`${opts.port != null ? `-p ${opts.port}` : ''} ` +
-		`${opts.username}@ssh.${opts.proxyUrl} ${opts.command}`
-	);
+	return [
+		...(opts.verbose ? ['-vvv'] : []),
+		'-t',
+		...['-o', 'LogLevel=ERROR'],
+		...['-o', 'StrictHostKeyChecking=no'],
+		...['-o', 'UserKnownHostsFile=/dev/null'],
+		...(opts.proxyCommand ? [opts.proxyCommand] : []),
+		...(opts.port ? ['-p', opts.port.toString()] : []),
+		`${opts.username}@ssh.${opts.proxyUrl}`,
+		opts.command,
+	];
```

Comment:
This function now returns an array of arguments instead of a string, which avoids the need for escaping arguments in a shell-specific way (cmd.exe _vs._ bash).